docs/advanced/theme.stories.mdx

```diff
@@ -7,7 +7,7 @@ import {
   BorderRadius,
   BorderWidth,
   Type,
-  MediaQueriesTable
+  MediaQueriesTable,
```

Comment:
A couple trailing commas were added to the MDX by my linter, I suspect this is based on the new Foundry v2 rules.

I've left them there but I'm happy to remove them if we want to lint the entire repo instead 🙂 Let me know what you think @connor-baer